.golangci.yml

```
@@ -119,3 +119,5 @@ linters-settings:
       - regexpMust
       - appendAssign
       - ifElseChain
+  unused:
+    local-variables-are-used: false
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)